json-formatter/CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ This document is formatted according to the principles of [Keep A CHANGELOG](htt
 
 * Process rules
 
+* Output background name
```

Comment:
We don't put a line between bullets in other changelogs - I'm not sure Markdown will render it as the same bullet list if you do?